src/util/RegistrationFormFactory.js

```diff
@@ -17,6 +18,72 @@ define([
 ], function (Okta, SchemaFormFactory, TextBox) {
 
   var _ = Okta._;
+  
+  var checkSubSchema = function(subSchema, value, model) {
+    var minLength = subSchema.get('minLength');
+    var maxLength = subSchema.get('maxLength');
+    var regex = subSchema.get('format');
+    
+    if (_.isNumber(minLength)) {
+      if (value.length < minLength) {
+        return false;
+      }
+    }
+    
+    if (_.isNumber(maxLength)) {
+      if (value.length > maxLength) {
+        return false;
+      }
+    }
```

Comment:
If ``minLength`` and ``maxLength`` are not numbers, do we want to fail or continue validation? Similar for the ``regex`` not being a string..

Reply:
Since these are coming from password policy API, they should follow the contract, but if they are not numbers/strings we'll skip that criteria.